transports/jms/src/test/java/org/mule/transport/jms/integration/JmsRequestRespondTestCase.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.transport.jms.integration;
+
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.mule.tck.functional.FlowAssert.verify;
+import org.mule.api.MuleMessage;
+
+import org.junit.Test;
+
+public class JmsRequestRespondTestCase extends AbstractJmsFunctionalTestCase
+{
+
+    @Override
+    protected String getConfigFile()
+    {
+        return "integration/jms-request-response.xml";
+    }
+
+
+    @Test
+    public void testNotSendingCorrelationIDWhenIsNotSettedUpTemporaryQueue() throws Exception
```

Comment:
testNotSendingCorrelationIDWhenIsNotSettedUpTemporaryQueue -> testNotSendingCorrelationIDWithTemporaryQueue (also fix english in other test names)